picturepyfect/get_property.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+def get_property(image):
+    """
+    Extract RGB or RGBA image properties. The output properties includes means and medians 
+    of RGB channels along with the file dimension and total pixels.
+    Parameters
+    ----------
+    image : numpy.ndarray
+        A n*n*3 numpy array to representing 3 or 4-channel RGB or RGBA image
+    Returns:
+    ---------
+    image_property: dictionary
+        a dictionary of image properties for dimension of width and height, total pixels,
+        and 3 channels' mean and median values separated by channel.
```

Comment:
Maybe add a comment about how the mean and median values are in a single list?

Reply:
Sure